File: app-kotlin/src/main/java/fudi/freddy/mokitosample/camera/CameraContract.java
package fudi.freddy.mokitosample.camera;

/**
 * @version 1.0
 * @autor Romell Domínguez
 * @date 8/13/17
 */

public interface CameraContract {

    interface CameraView{
        void openCamera(String path);
        void loadPicture(String bitmap);
        void showDefaultPicture();
    }
    interface CameraUserActions {
        void takePicture();
        void viewPicture();
        void viewDefaultPicture();
    }
}

File: app-kotlin/src/main/java/fudi/freddy/mokitosample/camera/CameraContract.kt
package fudi.freddy.mokitosample.camera

/**
 * @version 1.0
 * @autor Romell Domínguez
 * @date 8/13/17
 */
interface CameraContract {
    interface CameraView {
        fun openCamera(path: String?)
        fun loadPicture(bitmap: String?)
        fun showDefaultPicture()
    }

    interface CameraUserActions {
        fun takePicture()
        fun viewPicture()
        fun viewDefaultPicture()
    }
}